StatsdClient/StatsdClient.nuspec

```diff
@@ -3,7 +3,7 @@
   <metadata>
     <id>StatsdCsharpClient</id>
     <version>$version$</version>
-    <title>Simple Statsd Client for .Net 4.5</title>
+    <title>Simple Statsd Client for .Net 4.6.1</title>
```

Comment:
Can we just call this Simple StatsD Client

Reply:
Can't see why not. I will make the change in the next few days.